stripe/src/main/java/com/stripe/android/StripeFireAndForgetRequestExecutor.kt

```diff
@@ -53,17 +53,8 @@ internal class StripeFireAndForgetRequestExecutor : FireAndForgetRequestExecutor
     }
 
     override fun executeAsync(request: StripeRequest) {
-        val handlerThread = HandlerThread(
-            StripeFireAndForgetRequestExecutor::class.java.simpleName
-        )
-        handlerThread.start()
-        val handler = Handler(handlerThread.looper)
-        handler.post {
-            try {
-                execute(request)
-                handlerThread.quitSafely()
-            } catch (ignore: Exception) {
-            }
+        GlobalScope.launch {
```

Comment:
fwiw, i've been avoiding using global scope - https://medium.com/@elizarov/the-reason-to-avoid-globalscope-835337445abc

Reply:
check out https://stackoverflow.com/questions/52581809/how-to-use-coroutines-globalscope-on-the-main-thread as well

Reply:
```
However, GlobalScope.launch creates global coroutines. It is now developer’s responsibility to keep track of their lifetime.
```
```
If you're launching a coroutine from an Android event handler, you should use the current Activity as its coroutine scope. This will ensure your coroutine gets canceled when the activity gets destroyed. Without that the coroutine will go on, referring to the now-dead activity.
```

do we want to get this cancelled though?
also since this is a fire and forget request, isnt it fine to use GlobalScope?

Agreed that in pretty much any other scenario this would be an antipattern, but sounds like in this one it's probably fine

Reply:
> do we want to get this cancelled though?
> also since this is a fire and forget request, isnt it fine to use GlobalScope?

Yeah you're probably right, I wasn't too sure if we'd ever want it to get canceled or not. Sounds like its probably fine in this case.

Reply:
Thanks for the feedback, super helpful.